app/Tools/Features.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Ushahidi\App\Tools;
+
+class Features
+{
+
+	/**
+	 * @param array $config
+	 */
+	public function __construct(array $config)
+	{
+		$this->config = $config;
+	}
+
+	/**
+	 * Check if a feature is enabled
+	 * @param  string  $feature
+	 * @return boolean
+	 */
+	public function isEnabled($feature)
```

Comment:
It might be good to add a method we can call directly in our validators too. Future music though.